Added a note on CNV with OLM disconnected or proxy (#27467)
openshift/openshift-docs: virt/install/preparing-cluster-for-virt.adoc

To obtain an evaluation version of {product-title}, download a trial
from the {product-title} home page.
====

> * If deploying {VirtProductName} in a disconnected environment, you must configure xref:../../operators/admin/olm-restricted-networks.adoc[Operator Lifecycle Manager on restricted networks], or if using proxy xref:../../operators/admin/olm-configuring-proxy-support.adoc[configure proxy support in Operator Lifecycle Manager]

Reviewer: ```suggestion
* To deploy {VirtProductName} in a disconnected environment, you must xref:../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[configure Operator Lifecycle Manager on restricted networks].

* To use proxy with {VirtProductName}, you must xref:../../operators/admin/olm-configuring-proxy-support.adoc#olm-configuring-proxy-support[configure proxy support in Operator Lifecycle Manager].
```

Reviewer: I also recommend moving these bullet points to the top portion of this module (underneath `* You must manage your Compute nodes...` would probably work) instead of under the note about obtaining a trial of OCP.

Author: Fixed, thanks!
